src/api/app/views/shared/_package_branch_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<p>
+  <% if show_source_fields == true %>
+    <label for="linked_project"><strong>Name of original project:</strong></label><br/>
+    <%= text_field_tag 'linked_project', nil, :size => 80, :id => 'linked_project', required: true %><br/>
+    <label for="linked_package"><strong>Name of package in original project:</strong></label><br/>
+    <%= text_field_tag 'linked_package', nil, :size => 80, :id => 'linked_package', required: true %><br/>
+  <% end %>
+  <label for="target_package"><strong>Name of branched package in target project:</strong></label><br />
+  (If you leave this field blank, the original package name will be used) <br/>
+  <% if !package %>
```

Comment:
instead of this if with two cases, you could write like:

`<%= text_field_tag 'target_package', package.try(:name), :size => 80 %><br/>`